math/src/fft/errors.rs

```diff
@@ -9,8 +9,8 @@ use lambdaworks_gpu::cuda::abstractions::errors::CudaError;
 
 #[derive(Debug, Error)]
 pub enum FFTError {
-    #[error("Could not calculate {1} root of unity")]
-    RootOfUnityError(String, u64),
+    #[error("Could not calculate root of unity")]
```

Comment:
Would this be?
```suggestion
    #[error("Could not calculate {0} root of unity")]
```

Reply:
No, this is not a mistake, I'm limiting information because I removed thiserror, and it doesn't matter a lot

Reply:
Yes, we took a look at that library but it wasn't widely used. So we avoided it for security reasons